### PR TITLE
Adding in gradle build

### PR DESCRIPTION
**Pull request by justsomeshitiwrote**

### CODE REVIEW

File: ff4j-spring-boot-sample/build.gradle [hunk 1]
```diff
@@ -0,0 +1,67 @@
+buildscript {
+    ext.kotlin_version = '1.3.10' // Required for Kotlin integration
+    ext.spring_boot_version = '2.1.0.RELEASE'
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: I believe we can use the recent one which is `2.2.5.RELEASE`

File: ff4j-spring-boot-sample/build.gradle [hunk 2]
```diff
@@ -0,0 +1,67 @@
+buildscript {
+    ext.kotlin_version = '1.3.10' // Required for Kotlin integration
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: I believe we can use the recent one which is `1.3.70`

File: ff4j-spring-boot-sample/build.gradle [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    ext.spring_boot_version = '2.1.0.RELEASE'
+    ext.cucumber_version = '1.2.4'
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: For sample I don't think we use cucumber hence this dependency is not required.

File: ff4j-spring-boot-sample/build.gradle [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version" // Required for Kotlin integration
+    compile "org.thymeleaf:thymeleaf:$spring_boot_version"
+    compile "org.ff4j:ff4j-spring-boot-starter:1.8.2"
+    compile "org.ff4j:ff4j-web:1.8"
+    compile "org.springframework.boot:spring-boot-starter-thymeleaf:$spring_boot_version"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```
    compile "org.springframework.boot:spring-boot-starter-thymeleaf:$spring_boot_version"
```

Any reason why you have added this dependency ?

File: ff4j-spring-boot-sample/build.gradle [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    with jar
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version" // Required for Kotlin integration
+    compile "org.thymeleaf:thymeleaf:$spring_boot_version"
+    compile "org.ff4j:ff4j-spring-boot-starter:1.8.2"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```suggestion
    compile "org.ff4j:ff4j-spring-boot-starter:$ff4j.version"
```

You may want to externalise the property's and use the latest version

File: ff4j-spring-boot-sample/build.gradle [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    compile "org.ff4j:ff4j-web:1.8"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```suggestion
    compile "org.ff4j:ff4j-web:$ff4j.version"
```

You may want to externalise the property's and use the latest version

File: ff4j-spring-boot-sample/build.gradle [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    compile "org.springframework.boot:spring-boot-starter-thymeleaf:$spring_boot_version"
+    compile "org.springframework.boot:spring-boot-starter-web:$spring_boot_version"
+    compile "org.springframework.boot:spring-boot-starter-actuator:$spring_boot_version"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```
    compile "org.springframework.boot:spring-boot-starter-actuator:$spring_boot_version"
```

Why do we need this dependency ?

File: ff4j-spring-boot-sample/build.gradle [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    compile "io.micrometer:micrometer-registry-prometheus:1.1.0"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```
    compile "io.micrometer:micrometer-registry-prometheus:1.1.0"
```

Why do we need this dependency ?

File: ff4j-spring-boot-sample/build.gradle [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    testCompile "org.springframework.boot:spring-boot-starter-test:$spring_boot_version"
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]:
```
    testCompile "org.springframework.boot:spring-boot-starter-test:$spring_boot_version"
```

Why do we need this dependency ?

File: ff4j-spring-boot-sample/build.gradle (hunk 5 above)
Comment (justsomeshitiwrote) [reply ff4j-spring-boot-sample/build.gradle]: I tried using 1.8.3-SNAPSHOT but got was unable to resolve it.

File: ff4j-spring-boot-sample/build.gradle [hunk 10]
```diff
@@ -0,0 +1,60 @@
+buildscript {
+    ext.kotlin_version = '1.3.70' // Required for Kotlin integration
+    ext.spring_boot_version = '2.2.5.RELEASE'
+    ext.ff4j_version = '1.8.2'
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: you can use 1.8.4

File: ff4j-spring-boot-sample/build.gradle [hunk 11]
```diff
@@ -0,0 +1,60 @@
+buildscript {
+    ext.kotlin_version = '1.3.70' // Required for Kotlin integration
+    ext.spring_boot_version = '2.2.5.RELEASE'
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: you can use 2.2.6.RELEASE

File: ff4j-spring-boot-sample/build.gradle [hunk 12]
```diff
@@ -0,0 +1,60 @@
+buildscript {
+    ext.kotlin_version = '1.3.70' // Required for Kotlin integration
```
Comment (paul58914080) [inline ff4j-spring-boot-sample/build.gradle]: you can use 1.3.71

**Comment by justsomeshitiwrote** [comment]
@paul58914080 looks like I have addressed all the issues but I don't have the ability to merge this PR cna you?

**Pull request merged**